### PR TITLE
typings

### PR DESCRIPTION
**Pull request by gonzazoid**

### CODE REVIEW

File: src/common/badges/channelStatus.tsx [hunk 1]
```diff
@@ -18,7 +33,7 @@ export default class ChannelStatusStyle extends React.Component<any, any> {
     }
 
     render() {
-        const status = this.props.serviceStatus;
-        return <span className={`label label-table label-${this.classes[status]}`}>{status}</span>;
+        const {serviceStatus } = this.props;
```
Comment (drew2a) [inline src/common/badges/channelStatus.tsx]: redundant space

File: src/common/badges/jobName.tsx [hunk 2]
```diff
@@ -1,12 +1,17 @@
 import * as React from 'react';
 import { translate } from 'react-i18next';
 
-@translate('jobName')
-class JobStatus extends React.Component<any, any>{
+interface IProps {
+    t?: any;
```
Comment (drew2a) [inline src/common/badges/jobName.tsx]: `t' is bad name, it's speaks nothing

File: src/common/modalWindow.tsx [hunk 3]
```diff
@@ -2,11 +2,29 @@ import * as React from 'react';
...
 
-export default class ModalWindow extends React.Component<any, any> {
+interface IProps {
+    customClass: string;
+    modalTitle: string;
+    text: string;
+    visible?: boolean;
+    component: any;
+    copyToClipboard?: boolean;
+    // REFACTOR IT copyToClipboard component should receive child component
```
Comment (drew2a) [inline src/common/modalWindow.tsx]: may be it is better to transform this to TODO

File: src/common/logs/logsStack.tsx [hunk 4]
```diff
@@ -1,10 +1,14 @@
...
 
-export default class LogsStack extends React.Component <any, any> {
+interface IProps {
+    context: any;
+}
 
-    constructor(props: any) {
-        super(props);
-    }
+interface IState {
```
Comment (drew2a) [inline src/common/logs/logsStack.tsx]: Please, extract to a single file this multiple occurring interface

File: src/common/badges/jobName.tsx (hunk 2 above)
Comment (gonzazoid) [reply src/common/badges/jobName.tsx]: I agree. Unfortunately i18n library contributors by some reasons that I don't understand decided name it like that, so there is nothing  I can do with that - we just receive that function from `translate` HoC. The only one way to solve this problem is to write own HoC to rename that function, but in my opinion it's bad decision - `t` is quite well known name when we talking about internationalisation. Another wide used name for translate function is `_` (GNU gettext)

File: src/common/modalWindow.tsx (hunk 3 above)
Comment (gonzazoid) [reply src/common/modalWindow.tsx]: done

File: src/common/logs/logsStack.tsx (hunk 4 above)
Comment (gonzazoid) [reply src/common/logs/logsStack.tsx]: I've replaced them with inline empty interfaces `{}`

**Pull request merged**